synapseclient/models/__init__.py

```diff
@@ -13,11 +13,11 @@
 from synapseclient.models.services import FailureStrategy
 from synapseclient.models.table import (
     Column,
+    ColumnExpansionStrategy,
     ColumnType,
-    CsvResultFormat,
     FacetType,
-    Row,
     RowsetResultFormat,
```

Comment:
```ImportError: cannot import name 'RowsetResultFormat' from 'synapseclient.models.table' (/Users/tyu/sage/synapsePythonClient/synapseclient/models/table.py)```

Reply:
Try this again. I missed committing this change from my local